src/nohttp/whitelist.lines

```diff
@@ -0,0 +1,17 @@
+^http://[^/]*caucho.com.*
+^http://[^/]*gebish.org.*
+^http://[^/]*groovy-lang.org.*
+^http://[^/]*hsqldb.org.*
+^http://[^/]*sourceforge.net.*
```

Comment:
Why all of source forge?

Reply:
Several projects that we link to from the documentation are published on Source Forge without HTTPS access.

I just updated two of them in beb454a29952bc4f2fff533424c88105012039d6, but we still have to link via `http://` for libraries such as JiBX and HtmlUnitSource Forge.

Reply:
Or are you perhaps suggesting that we only (now) have explicit exclusions for `htmlunit.sourceforge.net` and `jibx.sourceforge.net`?

Reply:
Only having explicit exclusions sounds reasonable.

Reply:
Thanks for the feedback.

@rwinch, can you make this change to the PR?

Reply:
@sbrannen Done

Reply:
Thanks, Rob!